README.md

```diff
@@ -92,14 +92,21 @@ Your `~/.vimrc.local` might look like this:
 If you don't wish to install a vim plugin from the default set of vim plugins in
 `.vimrc.bundles`, you can ignore the plugin by calling it out with `UnPlug` in
 your `~/.vimrc.bundles.local`.
-blacklist in `~/.vimrc.ignorebundles`:
 
     " Don't install vim-scripts/tComment
     UnPlug 'tComment'
 
 `UnPlug` can be used to install your own fork of a plugin or to install a shared
 plugin with different custom options.
 
+    " Only load vim-coffee-script if a Coffeescript buffer is created
+    UnPlug 'vim-coffee-script'
+    Plug 'kchmck/vim-coffee-script', { 'for': 'coffee' }
```

Comment:
This seems like a good suggestion (whether we're `Unplug`ing or not).

Should take this approach for some of our the Plugs we include by default?